### ISSUE TITLE
Rate limiting changes for 1.8

### ISSUE THREAD

**Issue by Diman0** [Mailu/Mailu#1867]
Copied from #1582. For 1.8 we will for now increase rate limiting value and disable rate limiting for the  subnet. 

- Rate limiting
  - Document rate limiting
  - Currently the subnet is included in the rate limiting. This means that a user who repeatly fails to login the webmail, blocks the webmail for ALL users.

- For 1.8 and master
  - in mailu.env set the rate limit to a high value.
  - in mailu.env disable the rate limiter for the subnet.
  - And document this of course and change this in the documentation
  - Set status blocked on lubs pull request and request to further discuss this for mailu 1.9.
  -  Make authentication fast #1745 is a draft pr from nextgens which contains a solution for this problem.
  -  We need a new issue and PR for making these changes.

**Comment by Diman0** [comment]
Work to be done for the above:
- subnet rate limiting:
  - Change in docs/configuration.rst that AUTH_RATELIMIT_SUBNET is set to False by default.
  - Change in core/admin/mailu/internal/views/auth.py the default value of AUTH_RATELIMIT_SUBNET to False.
  - Triple check it is safe to backport this to 1.8. Otherwise this must be done in 2 separate PRs. One on branch 1.8 and one on branch master.

- higher default rate limit
  - Add to towncrier newsfragment that I must not forget to document this in the release notes (once 1.8 is released) that we increased the default value for rate limiting. 
  - Change in [config.html](https://github.com/Mailu/Mailu/blob/56a6821875a2742e18e087e681b97f6820edde9a/setup/templates/steps/config.html#L54) the default value of 10 to a high value.

**Comment by Diman0** [comment]
I will change the default value to 10000/minute . I think this is high enough to prevent any issues.
